database/migration/migrateData.ts

```diff
@@ -1451,9 +553,21 @@ const migrateData = async () => {
   }
   // Other version migrations continue here...
 
-  // Set indexes last:
-  console.log('Updating indexes...')
-  await DB.changeSchema(createIndexes)
+  // Update (almost all) Indexes, Views, Functions, Triggers regardless, since
+  // they can be dropped and recreated, or updated with no consequence:
+  const createIndexesScript = readFileSync(
+    path.join(getAppEntryPointDir(), '../database/buildSchema/49_index.sql'),
```

Comment:
Ah! Here, I think this file got missing from a commit. Not in this branch `49_index`

Reply:
Odd -- should be all good now.